net-p2p/biglybt/biglybt-3.8.0.2.ebuild

```diff
@@ -0,0 +1,128 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# for 3.8.0.2 USE=doc produces 2 errors:
+#   uis/src/com/biglybt/ui/swt/plugin/net/buddy/swt/BuddyPluginView.java:68:
+#   uis/src/com/biglybt/ui/swt/plugin/net/buddy/swt/BuddyPluginViewChat.java:45:
+#   error: package com.biglybt.ui.swt.plugin.net.buddy does not exist
+# no USE=test due to unsupported testing framework #839681
+JAVA_PKG_IUSE="source"
+MAVEN_PROVIDES="com.${PN}:${PN}-core:${PV} com.${PN}:${PN}-ui:${PV}"
+
+inherit desktop java-pkg-2 java-pkg-simple optfeature xdg
+
+DESCRIPTION="Feature-filled Bittorrent client based on Azureus"
+HOMEPAGE="https://www.biglybt.com"
+SRC_URI="https://github.com/BiglySoftware/BiglyBT/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/BiglyBT-${PV}"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64"
+
+SWT_SLOT="4.34"
+CP_DEPEND="
+	dev-java/commons-cli:0
+	dev-java/swt:${SWT_SLOT}[cairo]
+"
+
+# Due to removal of AENameServiceDescriptor it would fail to start under jdk:1.8
+# StartServer ERROR: unable to bind to 127.0.0.1:6880 listening for passed torrent info: \
+# sun.net.spi.nameservice.NameServiceDescriptor: Provider com.biglybt.core.util.spi.AENameServiceDescriptor not found
+#
+# Also it fails with >=jdk-23 so we limit the max version #936549
+# NOTE: BiglyBT works with [headless-awt]
+DEPEND="${CP_DEPEND}
+	|| (
+		virtual/jdk:11
+		virtual/jdk:17
+		virtual/jdk:21
+	)
+"
+RDEPEND="${CP_DEPEND}
+	>=virtual/jre-1.8:*
+"
+
+DOCS=( README.md ChangeLog.txt )
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-3.2.0.0-disable-SWTUpdateChecker.patch
+	"${FILESDIR}"/${PN}-3.2.0.0-disable-shared-plugins.patch
+	"${FILESDIR}"/${PN}-3.5.0.0-Entities.javadoc.patch
+	"${FILESDIR}"/${PN}-3.6.0.0-disable-PluginUpdatePlugin.patch
+	"${FILESDIR}"/${PN}-3.6.0.0-fix-ambiguous.patch
```

Comment:
If you find someone to merge this, I won't mind.  But personally, I prefer cleartexting instead of using`${PN}`.
And I don't understand why FILESDIR is quoted but PN is not.